### DISCUSSION TITLE
Downloaded log archive zip contents now all have bad dates

### DISCUSSION THREAD

**Discussion by BrianInglis** [community/community#119958] [Actions]
### Select Topic Area

Bug

### Body

Downloaded log archive zip contents now all have bad dates - looks like Windows date "-1", which if like Unix functions, may indicate an error:
```
$ unzip -lv ~/Downloads/logs_22838092776.zip
Archive:  .../Downloads/logs_22838092776.zip
 Length   Method    Size  Cmpr    Date    Time   CRC-32   Name
--------  ------  ------- ---- ---------- ----- --------  ----
  224927  Defl:N    32793  85% 00-00-1980 00:00 968fd732  0_source _ build.txt
  134725  Defl:N    21379  84% 00-00-1980 00:00 b094f331  1_x86_64 _ build.txt
  227861  Defl:N    33641  85% 00-00-1980 00:00 ba12ae33  2_noarch _ build.txt
--------          -------  ---                            -------
  587513            87813  85%                            3 files
$ TZ=UTC unzip -oq ~/Downloads/logs_22838092776.zip
$ TZ=UTC ls -go --full *build.txt
-rw-r--r-- 1 224927 1979-12-31 00:00:00.000000000 +0000 '0_source _ build.txt'
-rw-r--r-- 1 134725 1979-12-31 00:00:00.000000000 +0000 '1_x86_64 _ build.txt'
-rw-r--r-- 1 227861 1979-12-31 00:00:00.000000000 +0000 '2_noarch _ build.txt'
```

**Comment by grandmas-favorite** [comment]
Hi @BrianInglis!

Would you be able to provide a little more context about your use case and the details around where and how you downloaded the log archive?

I think it would help myself and other Community members understand the issue better!

**Reply by BrianInglis** [reply]
@cygwin/scallywag CI builds source and arch packages when package build script repo pushed by packager; 
select Actions/scallywag/run/Job summary; 
select arch/build/log; 
select top right gear icon drop down: 
select Download log archive; 
downloads zip with all logs as shown above. 
Only source and one of noarch or x86_64 has significant content: longer running; other is dummy run. 
Workflow maintainer says logs and zips are produced by run infrastructure.
